src/libs/PusherConnectionManager.js

```diff
@@ -16,16 +18,27 @@ function init() {
     }));
 
     /**
-     * Events that happen on the pusher socket are used to determine if the app is online or offline.
-     * The offline setting is stored in Onyx so the rest of the app has access to it.
-     *
      * @params {string} eventName
      */
-    Pusher.registerSocketEventCallback((eventName, data) => {
+    Pusher.registerSocketEventCallback((eventName, error) => {
+        const errorType = lodashGet(error, 'type');
+        const code = lodashGet(error, 'data.code');
```

Comment:
If these are only used in `case 'error'` should we put them down there?

Reply:
I can make this change. eslint was complaining about this, but I think the solution is to use brackets to make the "lexical scoping" clearer.
![2022-04-22_06-16-02](https://user-images.githubusercontent.com/32969087/164754568-5b22de54-acfc-4ef4-a5f9-014ade21cf2a.png)

Reply:
Interesting, I didn't know it would complain about that - nice fix though 👍